components/schemes/Container/src/MDCContainerScheme.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "MaterialColorScheme.h"
+#import "MaterialShapeScheme.h"
+#import "MaterialTypographyScheme.h"
+
+/**
+ A class conforming to the MDCContainerScheming provides scheme properties for all the
+ theming systems (color, typography, shape, etc.) MDC supports.
+ */
```

Comment:
Looks like the wrong documentation. This is written for the interface, but was attached to the protocol declaration.

Reply:
Addressed by stating MDCContainerScheming explicitly as a protocol.

Reply:
Addressed in da30204eb15d8e8b46631210c33fcb1e74cc13ee.